### ISSUE TITLE
Not running on Mac

### ISSUE THREAD

**Issue by MatthewFrench** [electron/electron-quick-start#51]
Here's the output: 

```text
> Matthews-MacBook-Pro:~ Matt$ cd /Users/Matt/Downloads/electron-quick-start-master 
> Matthews-MacBook-Pro:electron-quick-start-master Matt$ npm install && npm start
> -
> > electron-prebuilt@0.37.8 postinstall /Users/Matt/Downloads/electron-quick-start-master/node_modules/electron-prebuilt
> > node install.js
> 
> electron-prebuilt@0.37.8 node_modules/electron-prebuilt
> ├── extract-zip@1.5.0 (debug@0.7.4, mkdirp@0.5.0, yauzl@2.4.1, concat-stream@1.5.0)
> └── electron-download@2.1.2 (path-exists@1.0.0, home-path@1.0.3, minimist@1.2.0, debug@2.2.0, mkdirp@0.5.1, rc@1.1.6, mv@2.1.1, nugget@1.6.2)
> 
> > electron-quick-start@1.0.0 start /Users/Matt/Downloads/electron-quick-start-master
> > electron main.js
> 
> App threw an error when running [TypeError: Cannot read property 'on' of undefined]
```

**Comment by jlord** [comment]
Have you npm installed a module called `electron` before (is it inside of your `node_modules` folder in this project)? If so, can you delete that folder and try running again?

Possibly a dupe of #53

**Comment by kevinsawicki** [comment]
I believe this should be fixed in Electron 1.1.2, closing this out.
